tgui/packages/tgui/interfaces/Wizard_Spellbook.js

```diff
@@ -105,32 +105,54 @@ const Spell = (props, context) => {
   const { spell, category } = props;
 
   const [purchased, setPurchased] = useSharedState(context, spell+"p", false);
-  let spell_contents = []; // Non-associated list of: desc, cost, cooldown, vr_allowed
+  let spell_contents = []; // Non-associated list of: desc, cost, cooldown, vr_allowed, spell_img
   for (let spell_data in spellbook_contents[category][spell]) {
     spell_contents.push(spellbook_contents[category][spell][spell_data]);
   }
 
   return (
     <Stack.Item>
-      <Section>
+      <Section mt={0.5} mb={-1.5}>
         {(vr === 1) && (spell_contents[3] === 0) && (
-          <Dimmer>
+          <Dimmer mb={3} mt={-2}>
             <Box fontSize={1.5} backgroundColor={"#384e68"} p={2}>
               Spell unavailable in VR
             </Box>
           </Dimmer>
         )}
         <Section
-          title={spell}
-          buttons={
-            <Flex>
-              <Button
-                backgroundColor={"green"}
-                disabled={spell_slots < spell_contents[1] || purchased}
-                onClick={() => { setPurchased(true); act("buyspell", { spell: spell }); }}
-              >
-                {If_Purchased_Text(purchased, spell_contents[1], spell_slots)}
-              </Button>
+          title={
+            <Flex justify={"space-between"} align={"end"}>
```

Comment:
```suggestion
            <Flex justify="space-between" align="end">
```

Reply:
Just curious, why should I do this? I don't think there is any difference and its the first time I'm seeing an apprehension towards {}

Reply:
Mostly just a case of "it's the preferred syntax for representing static strings in JSX". Same sort of thing for `foo={true}` vs. just adding `foo` (without anything else) for boolean true things.

Not requiring this change, just encouraging the convention.

Reply:
Ah that makes sense, I am fine with that convention then and I will try to keep it in mind.